vignettes/articles/data_from_epireview.Rmd

```diff
@@ -0,0 +1,270 @@
+ebola_si_epidist <- as_epidist(ebola_si)
+ebola_si_epidist
+```
+
+With the probability distribution for the serial interval we can utilise some of the `<epidist>` methods. Here we illustrate this by checking that the `<epidist>` is parameterised, plotting the PDF and CDF, and generating 10 random numbers sampling from the distribution.
+
+```{r, is-epidist-parameterised}
+is_parameterised(ebola_si_epidist)
+```
+
+```{r, plot-epidist}
+plot(ebola_si_epidist)
+```
+
+```{r, generate-epidist}
+generate(ebola_si_epidist, times = 10)
+```
+
+## Specifying the probability distribution if unknown
+
+There may be instances where a delay distribution is reported, but either a probability distribution is not fit to the data, or it is not reported which probability distribution the parameters correspond to. In these cases, and when a parametric probability distribution is required for a particular epidemiological task then assuming a probability distribution can be useful. 
+
+::: {.alert .alert-danger}
+**Please use this feature with caution**. Assuming an incorrect probability distribution and applying this in an epidemiological method can lead to erroneous results. Additionally, if a probability distribution is specified by the user it will overwrite any probability distribution specified in input data (e.g. {epireview} parameter data) which can lead to an error if the distribution name supplied and parameters input are incompatible See `?as_epidist` details for more information.
+:::
+
+Just as the example above we will load the Ebola parameters using the `epireview::load_epidata()` function and subset to just the parameters (`$params`).
+
+```{r, load-and-subset-ebola}
+ebola_data <- load_epidata("ebola")
+ebola_params <- ebola_data$params
+```
+
+Here we will use the serial interval for Ebola reported by @fayeChainsTransmissionControl2015a. This is stored, over two rows of the parameter table, as the mean and standard deviation, but there is no probability distribution specified. 
+
+The code chunk below subsets the Ebola parameter table to just return the serial interval from @fayeChainsTransmissionControl2015a.
+
+```{r, subset-ebola-serial-interval}
+ebola_si <- ebola_params[
+  which(
+    grepl(pattern = "Faye", x = ebola_params$article_label, fixed = TRUE) &
+      grepl(pattern = "serial", ebola_params$parameter_type, fixed = TRUE)
+  ),
+]
+```
+
+If we were to supply this data to `as_epidist()` we would get an unparameterised `<epidist>` object because no probability distribution is stated.
+
+```{r, convert-to-epidist-unspecified-prob-dist}
+ebola_si_epidist <- as_epidist(ebola_si)
+ebola_si_epidist
+is_parameterised(ebola_si_epidist)
+```
+
+Given that we can convert the mean and standard deviation into parameters of a probability distribution if we assume a distribution form, we can supply this data to `as_epidist()`.
+
+```{r, convert-to-epidist-assumed-prob-dist}
+ebola_si_epidist <- as_epidist(ebola_si, prob_dist = "gamma")
```

Comment:
so what's going on here is that the functions to convert the summary statistics into parameters are being used?
I would add how this is being done to the description in line 248

Reply:
Yes, if the probability distribution is assumed we can convert the mean and sd to the distribution parameters using the {epiparameter} conversion functions internally.

I'll add a description to the text.

Reply:
Resolved in 9d2594430130da7bde930d740d5bc368369b47d5.